### DISCUSSION TITLE
There is a way to add Auto HDR for AutoActions?

### DISCUSSION THREAD

**Discussion by kolunio82** [Codectory/AutoActions#107] [Ideas]
Hi. In windows 11 I can use auto HDR feature which is useful in many games. But not in all games auto HDR looks property. So can we please have a option to enable auto HDR before select exe start and to disable after exit? It would be very helpful and useful.

Thanks

**Comment by CaveWaverider** [comment]
If it's possible, it'd certainly be nice to have in Auto Actions to have everything in one tool, but I'm not sure if it's within the scope of this program.

In the meantime, you can do so in Windows 11 Settings tself. Go to Settings->Display Settings->Graphics use Browse to pick a game/app (either UWP or Desktop), click on the game on the list, click Options and tick "Don't use Auto HDR".

**Reply by kolunio82** [reply]
Yes I know that. But would be nice to do everything in one app. It takes a lot of time to uncheck games from auto HDR. If we would have auto HDR in AutoActions life would be much easier. 😊
